addons/danger/functions/fnc_assaultVehicle.sqf

```diff
@@ -45,7 +45,7 @@ private _pos = AGLToASL ((selectRandom _buildings) vectorAdd [0.5 - random 1, 0.
 _unit doSuppressiveFire _pos;
 
 // shoot cannon
-_cannon = random 1 > 0.2;
+private _cannon = random 1 > 0.2;
```

Comment:
Hmm. This check is more advanced in v1.4.1.  In fact the entire function. Guess it missed something crossing over. Will update after this cleanup is merged. 

Also: https://github.com/nk3nny/LambsDanger/issues/24